sw/device/silicon_creator/manuf/base/BUILD

```diff
@@ -2,6 +2,8 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+load("@//rules:signing.bzl", "offline_presigning_artifacts", "offline_signature_attach")
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
```

Comment:
Using this?

Reply:
Ah looks like those just got moved around as a part of `bazel run //quality:buildifier_fix` running. Not sure why CI didn't catch that before. They were already in place though.